src/Type/PostObject/Connection/PostObjectConnectionResolver.php

```diff
@@ -366,7 +366,7 @@ public static function sanitize_input_fields( array $args, $source, array $all_a
 		 * @since 0.0.5
 		 * @return array
 		 */
-		$query_args = apply_filters( 'graphql_map_input_fields_to_wp_query', $query_args, $args, $source, $all_args, $context, $info );
+		$query_args = apply_filters( 'graphql_map_input_fields_to_wp_query', $query_args, $args, self::$post_type, $source, $all_args, $context, $info );
```

Comment:
If we add `self::$post_type` to the end, after `$info` this wouldn't be a breaking change. 

As in, anyone with existing code using this filter wouldn't break. I think this is a good addition to the filter, let's just add it as the 7th param instead of the 3rd.

Reply:
```suggestion
		$query_args = apply_filters( 'graphql_map_input_fields_to_wp_query', $query_args, $args, $source, $all_args, $context, $info, self::$post_type );
```

Reply:
also would need to update the DocBloc